app/templates/courses/prepaid-activation-codes-modal.pug

```diff
@@ -0,0 +1,126 @@
+extends /templates/core/modal-base-flat.jade
```

Comment:
I think its better to write new Modals in Vue if possible

Reply:
😂 good... point